package.json

```diff
@@ -2,27 +2,25 @@
   "name": "@cowprotocol/cow-sdk",
   "version": "1.0.2-RC.0",
   "license": "(MIT OR Apache-2.0)",
-  "source": "src/index.ts",
-  "main": "./dist/index.js",
-  "module": "./dist/index.module.js",
-  "exports": {
-    "require": "./dist/index.js",
-    "default": "./dist/index.modern.mjs"
-  },
-  "types": "dist/index.d.ts",
   "files": [
     "/dist"
   ],
   "scripts": {
-    "build": "microbundle -f modern,esm,cjs",
-    "start": "microbundle -f modern,esm,cjs watch",
+    "prebuild": "rm -rf dist && yarn run test:coverage && yarn run make-badges && yarn run codegen",
+    "build": "tsc --module commonjs",
+    "postbuild": "cp package.json dist && cp README.md dist",
+    "start:metadata": "tsc-watch --noClear -p ./tsconfig.json --onSuccess \"node ./dist/metadata/index.js\"",
+    "start:order-book": "tsc-watch --noClear -p ./tsconfig.json --onSuccess \"node ./dist/order-book/index.js\"",
+    "start:subgraph": "tsc-watch --noClear -p ./tsconfig.json --onSuccess \"node ./dist/subgraph/index.js\"",
     "lint": "eslint src",
     "format": "prettier --write \"src/**/*.+(ts|json)\"",
     "test": "jest",
-    "test:coverage": "jest --coverage --json --outputFile=jest.results.json && cat ./coverage/lcov.info | coveralls",
-    "prepare": "npm run build && npm run graphql:codegen",
+    "test:coverage": "jest --collectCoverage",
+    "codegen": "npm run graphql:codegen && npm run swagger:codegen",
     "prepublishOnly": "npm test && npm run lint",
-    "graphql:codegen": "graphql-codegen --config graphql-codegen.yml"
+    "make-badges": "istanbul-badges-readme",
```

Comment:
I didn't see this change mentioned anywhere. Shouldn't it deserve more attention?

Reply:
Reverted it